app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comments < ApplicationRecord
+  belongs_to :post
+  belongs_to :user
+
+  def update_post_comment_count
+    post.update(comments_counter: post.comments.count)
+  end
+end
```

Comment:
[OPTIONAL] It is a good practice to use rails callbacks in scenarios like this. This will help keep your code DRY.

Reply:
Thanks for the review @fobadara, It would have been great to have a code example provided.